packages/server/modules/notifications/services/handlers/mentionedInComment.ts

```diff
@@ -151,7 +151,7 @@ function buildEmailTemplateParams(
     objectId,
     commitId
   })
-  const url = new URL(commentRoute, getBaseUrl()).toString()
+  const url = new URL(commentRoute, getFrontendOrigin()).toString()
```

Comment:
mention email cta links were busted locally (only where FE and BE origin differs afaik)